src/main/java/io/vertx/core/http/impl/ClientConnection.java

```diff
@@ -74,6 +76,11 @@
   private HttpClientRequestImpl requestForResponse;
   private WebSocketImpl ws;
 
+  static {
+    CONNECTION_CLOSED_EXCEPTION = new VertxException("Connection was closed");
+    CONNECTION_CLOSED_EXCEPTION.setStackTrace(new StackTraceElement[0]);
```

Comment:
I'm a bit worried by the absence of stack trace in the exception. Could you give a reason for this ?
(I understand that it is required to build the singleton)